src/js/actions/ui.js

```diff
@@ -349,27 +349,51 @@ define(function (require, exports) {
         _displayConfigurationChangedHandler;
 
     /**
+     * @const
+     * @type {number}
+     */
+    var EVENT_DEBOUNCE_DELAY = 200;
+
+    /**
      * Register event listeners for UI change events, and initialize the UI.
      *
      * @return {Promise}
      */
     var beforeStartup = function () {
-        var DEBOUNCE_DELAY = 200;
-
-        var setTransformDebounced = synchronization.debounce(function (event) {
-            if (event.transform) {
-                return this.flux.actions.ui.setTransform(event.transform);
-            }
-        }, this, DEBOUNCE_DELAY, false);
+        var scrolling = false,
+            updateTransformDebounced = synchronization.debounce(function () {
+                return this.flux.actions.ui.updateTransform()
```

Comment:
I changed this from `setTransform` to `updateTransform` because at least some of the artboard overlay bugs were due to `set`ing an old transform right after a previous call to `updateTransform`. Currently `setTransform` (which just updates the model with a given transform, typically from an event handler) is dead code. I'm open to removing it, but I thought I'd check with someone else before I did that.